docs/cave/goes-satellite.md

```diff
@@ -0,0 +1,308 @@
+# GOES 16/17
+
+The ***goesr*** EDEX decoder supports the ingest of GOES products coming over NOAAPort and Unidata's IDD. These include [**single channel imagery**](#individual-channels), [**derived products**](#derived-products) (Level 2b netCDF files), gridded [**Geostationary Lightning Mapper**](#geostationary-lightning-mapper-glm) (GLM) products (produced by Eric Bruning at Texas Tech), CIRA created [**RGB**](#cira-cloud-snow) specific products, and [**vertical temperature/moisture profiles**](#vertical-temperature-and-moisture-profile). Using derived parameters, additional [**RGB**](#rgb-composites) and [**channel difference**](#channel-differences) products can be loaded. The ***dmw*** EDEX decoder supports the ingest of GOES [**derived motion winds**](#derived-motion-winds).
```

Comment:
This looks a lot better!  I just think I'd make [**CIRA created RGB**] the whole link to the cira products, so it's a little more clear (and the link is easier to see).

Reply:
Also, do you want to change the link to point to the cira geo color, since that is the first cira product shown below?